Xamarin.Forms.Xaml.UnitTests/DataTemplateExtension.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+
+using Xamarin.Forms.Core.UnitTests;
+
+namespace Xamarin.Forms.Xaml.UnitTests
+{
+	public partial class DataTemplateExtension : ContentPage
```

Comment:
This is just me, but there are too many things called `DataTemplateExtension` here... both the library types, the page, the test.

Reply:
too many, but no confusion